restic/README.md

```diff
@@ -0,0 +1,42 @@
+# restic
+
+restic is a backup program that is fast, efficient and secure.
+
+For detailed usage check out the [documentation](https://restic.readthedocs.io/en/latest).
+
+## Maintainers
+
+* The Habitat Maintainers: <humans@habitat.sh>
+
+## Type of Package
+
+Binary package
+
+## Usage
+
+Start off with creating a repository for your backups:
+
+```console
```

Comment:
I picked that up in restic's docs actually, I found that it's an alias for bash in GitHub's supported grammars

IMHO it seems a bit more appropriate when illustrating a prompt + output, since the content isn't something you could run with bash

Reply:
Since I started using `console` I've seen some formatters color bash syntax on `$` lines and grey out the rest as output